mod_reforged/hooks/skills/backgrounds/slave_background.nut

```diff
@@ -39,4 +37,43 @@
 				return 0.25;
 		}
 	}}.getPerkGroupMultiplier;
+
+	q.onChangeAttributes = @() { function onChangeAttributes()
+	{
+		local c = {
+            Hitpoints = [
+                -10,
+                -10
+            ],
+            Bravery = [
+                -5,
+                0
+            ],
+            Stamina = [
+                -5,
+                5
+            ],
+            MeleeSkill = [
+                -10,
+                10
+            ],
+            RangedSkill = [
+                -15,
+                15
+            ],
+            MeleeDefense = [
+                -10,
+                10
+            ],
+            RangedDefense = [
+                -10,
+                10
+            ],
+            Initiative = [
+                -15,
+                15
```

Comment:
This is a buff of +5 average Initiative for Slaves (from -5 to +-0). Is this intended? 
You dont buff their average in any other attribute.

Perhaps you want the range to be -20, 10 instead?